frontend/app/src/components/helper/BinancePairsSelector.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-card v-bind="$attrs">
+    <div class="mx-4 pt-2">
+      <v-autocomplete
+        :items="allMakerts"
+        :filter="filter"
+        :search-input.sync="search"
+        multiple
+        :loading="loading"
+        :disabled="loading"
+        hide-details
+        hide-selected
+        hide-no-data
+        return-object
+        chips
+        clearable
+        :outlined="outlined"
+        :open-on-clear="false"
+        :label="label ? label : $t('blockchain_account_selector.default_label')"
```

Comment:
Shouldn't the label be replaced with something like
"Trade markets"? (extracted to the en.json)